docs/clients/p4rt-ctl.rst

```diff
@@ -91,6 +91,24 @@ services that are available for a p4runtime client.
 For information on security and enabling TLS on gRPC ports, refer to
 guides/security-guide.md
 
+Usage: p4rt-ctl [OPTIONS] COMMAND [ARG...]
+positional arguments:
+  command               Subcommand to run
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -g, --grpc_addr       GRPC_ADDR P4Runtime gRPC server address, format : <server IP>:<port>
+
+Details about supported option (``-g`` GRPC_ADDR)::
+---------------------------------------------------
+This is an optional parameter and default value is 127.0.0.1:9559. If P4Runtime server is not running
+locally, user can specify the address using this option.
+
+Eg: If server is listening on ``5.5.5.5:9559``, command to set pipeline will be:
```

Comment:
```suggestion
For example, if the server is listening on ``5.5.5.5:9559``, the command to set the pipeline will be:
```
_e.g._ is the abbreviation of the Latin expression _exempli gratia_, which means "for instance" or "for example." It should not be used to start a sentence.